app/src/main/java/agency/five/codebase/android/movieapp/ui/component/UserScoreProgressBar.kt

```diff
@@ -0,0 +1,66 @@
+package agency.five.codebase.android.movieapp.ui.component
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.StrokeCap
+import androidx.compose.ui.graphics.drawscope.Stroke
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.Dp
+import androidx.compose.ui.unit.TextUnit
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import kotlin.math.absoluteValue
+
+@Composable
+fun UserScoreProgressBar(
+    userScore: Float,
+    modifier: androidx.compose.ui.Modifier = androidx.compose.ui.Modifier,
+    fontSize: TextUnit = 15.sp,
+    radius: Dp = 42.dp,
```

Comment:
We should not expose the circle radius to the outside caller.
The radius is always the half of the view width (occupying the maximum possible area of the view).